### PR TITLE
Docs for getting started

### PR DESCRIPTION
**Pull request by perolavsvendsen**
Adding a section to the docs explaining how to get started and how to set up FMU workflow to use fmu-dataio. This section is referenced from the Sumo documentation.

### CODE REVIEW

File: docs/examples.rst [hunk 1]
```diff
@@ -3,9 +3,6 @@ Examples
 
 In the following is a list of examples you can study and copy.
```
Comment (rnyb) [inline docs/examples.rst:4]: use  `collection` instead of `list?`

File: docs/examples.rst [hunk 2]
```diff
@@ -3,9 +3,6 @@ Examples
...
 
-First is a snippet of the ``global_variables.yml`` file which holds the static
-metadata. In real cases this file will be much longer.
-
 Then there are several examples for various datatypes. In general there is a python
 script, and next is an example from a metadata file that is produced by
 the script.
```
Comment (rnyb) [inline docs/examples.rst:8]: Rephrase slightly? E.g.
`The examples covers various datatypes. The examples typically shows a python script, together with a corresponding metadata file that would be produced when running the script.`

File: docs/examples.rst [hunk 3]
```diff
@@ -17,8 +14,9 @@ can comment it out in your code.
 The global variables used here
 ------------------------------
 
-Here are the relevant sections in the global variables (output) file (press plus to
-expand):
+This is a snippet (relevant sections only - in real life, this is much longer) of the
+``global_variables.yml`` file which holds the static metadata described in the 
+`previous section <./preparations.html>`__. In real cases this file will be much longer.
```
Comment (rnyb) [inline docs/examples.rst:20]: this is much longer ... will be much longer. Is it enough to mention this once?

**Review by rnyb — CHANGES_REQUESTED** [review]
Added some minor text edit suggestions you can consider

**Comment by perolavsvendsen** [comment]
> Added some minor text edit suggestions you can consider

Nice, thanks - updated 👍

**Review by rnyb — APPROVED** [review]

**Comment by perolavsvendsen** [comment]
Bah... This didn't work. Worked fine when built locally, so I don't know. Perhaps @jcrivenaes can spot my mistake. Apparently, `/preparations.html` does not build on ReadTheDocs.

**Pull request merged**